src/rest-server/src/models/v2/job/k8s.js

```diff
@@ -1093,7 +1093,10 @@ const generateExitDiagnostics = (diag) => {
 };
 
 const extractRuntimeOutput = (podCompletionStatus) => {
-  if (_.isEmpty(podCompletionStatus)) {
+  if (
+    _.isEmpty(podCompletionStatus) ||
```

Comment:
what do you want to check for `podCompletionStatus`, isn't `if (podCompletionStatus)` enough

Reply:
Just want to check for `null` or `undefined`. Will change to use simple `if` sentence

Reply:
Fixed